packages/react-table/src/demos/table-demos/Compact.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import {
+  Button,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+  Pagination,
+  Select,
+  SelectVariant,
+  SelectOption,
+  PageSection
+} from '@patternfly/react-core';
+import { TableComposable, Thead, Tr, Th, Tbody, Td, ActionsColumn } from '@patternfly/react-table';
+
+import FilterIcon from '@patternfly/react-icons/dist/esm/icons/filter-icon';
+import CheckIcon from '@patternfly/react-icons/dist/esm/icons/check-icon';
+import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
+
+export const ComposableTable = () => {
+  const [isSelectOpen, setIsSelectOpen] = React.useState(false);
+
+  const columns = ['Contributor', 'Position', 'Location', 'Last seen', 'Numbers', 'Icons'];
+  const rows = [
+    ['Sam Jones', 'CSS guru', 'Not too sure', 'May 9, 2018', '0556'],
+    ['Amy Miller', 'Visual design', 'Raleigh', 'May 9, 2018', '9492'],
+    ['Steve Wilson', 'Visual design lead', 'Westford', 'May 9, 2018', '9929'],
+    ['Emma Jackson', 'Interaction design', 'Westford', 'May 9, 2018', '2217']
+  ];
+
+  const defaultActions = () => [
+    {
+      title: 'Settings',
+      // eslint-disable-next-line no-console
+      onClick: () => console.log(`clicked on Settings`)
+    },
+    {
+      title: 'Help',
+      // eslint-disable-next-line no-console
+      onClick: () => console.log(`clicked on Help`)
+    }
+  ];
+
+  const renderPagination = (variant, isCompact) => (
+    <Pagination
+      isCompact={isCompact}
+      itemCount={36}
+      page={1}
+      perPage={10}
+      onSetPage={(_evt, value) => {
+        this.fetch(value, 10);
```

Comment:
I don't think you have a data source hooked up, so when I interact with the pagination component, I am getting an error when it tries to fetch. Maybe you should use something like a console log in these event handlers rather than a fetch to avoid the error or any possible confusion?

Reply:
I removed these event handlers -- as we discussed in standup -- since there isn't enough data to paginate here anyway